src/view.tree.js

```diff
@@ -194,7 +195,7 @@ class TreeView {
     }
   }
 
-  syncSelection(repo) {
+  async syncSelection(repo) {
```

Comment:
Miss update at  https://github.com/ovity/octotree/blob/cc60d85da02a5f817f1b4c222207779398f94280/src/main.js#L146

Reply:
Thanks, i totally missed this